scripts/add_attribute.py

```diff
@@ -111,7 +124,8 @@
                 attrName = line[12:-36]
             if not line or attrName > attributeName:
                 newLines.append("        if \"" + attributeName + "\" in attributes:  # pragma no branch")
-                newLines.append("            assert attributes[\"" + attributeName + "\"] is None or isinstance(attributes[\"" + attributeName + "\"], " + attributeAssertType + "), attributes[\"" + attributeName + "\"]")
+                if attributeAssertType:
+                    newLines.append("            assert attributes[\"" + attributeName + "\"] is None or isinstance(attributes[\"" + attributeName + "\"], " + attributeAssertType + "), attributes[\"" + attributeName + "\"]")
```

Comment:
Same here

Reply:
Right - this is the code that generates the other code you were commenting on.

Reply:
Note that `attributeAssertType` may have its purposes, so I am just going to remove that field from the simple attribute types.